keel-sql/src/main/resources/db/changelog/20200522-standardize-timestamps.yml

```diff
@@ -1,39 +1,153 @@
 databaseChangeLog:
 - changeSet:
-    id: event-uid-timestamp-index
+    id: standardize-timestamps-agent_lock
```

Comment:
Did you rollback the migration in prestaging? If not, won’t reusing the migration filename cause a conflict when liquibase tries to run it again?

Reply:
I did roll it back.